src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -591,4 +615,33 @@ protected function note($message)
             $this->output->writeln($message);
         }
     }
+
+    /**
+     * Set the event dispatcher instance.
+     *
+     * @param  \Illuminate\Contracts\Events\Dispatcher  $dispatcher
+     * @return void
+     */
+    public function setEventDispatcher(Dispatcher $dispatcher)
+    {
+        static::$dispatcher = $dispatcher;
+
+        return $this;
```

Comment:
Your method is void but you're returning something here. Remove this return because it's not needed.

Reply:
Done